Formula/easy-rsa.rb

```diff
@@ -0,0 +1,34 @@
+class EasyRsa < Formula
+  desc "CLI utility to build and manage a PKI CA."
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/3.0.1.tar.gz"
+  sha256 "a1fff75a27ea7da3f37fbfed715633f55b9ca25f5b14cac38e525c5c995e68ae"
+
+  option "with-openssl", "Use the openSSL provided by homebrew"
+  depends_on "openssl" => :optional
+
+  def install
+    # We don't use the installer from the package, as it doesn't organize things
```

Comment:
The package installer leaves the easyrsa command in the prefix, not in a bin/ subdirectory. I re-laid it out in here so that the tool can find all of its data properly, and end up on the homebrew PATH